ci-operator/config/openshift/release/openshift-release-master__ci-4.20.yaml

```diff
@@ -366,6 +366,23 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-upgrade-vsphere-runc
+- as: e2e-gcp-ovn-dra
+  interval: 24h
+  steps:
+    cluster_profile: gcp
+    env:
+      FEATURE_GATES: '["DynamicResourceAllocation=true", "DRAAdminAccess=true"]'
+      FEATURE_SET: CustomNoUpgrade
+      GPU_COMPUTE_NODE_REPLICAS: "1"
+      GPU_COMPUTE_NODE_TYPE: a2-highgpu-1g
+      OPENSHIFT_SKIP_EXTERNAL_TESTS: "true"
+      TEST_ARGS: --run=\[Feature:DynamicResourceAllocation\] --max-parallel-tests=1
+      TEST_SUITE: all
+      TEST_TYPE: suite
```

Comment:
Looks like this could be omitted as the default value of the env var is already "suite"
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml#L16-L17